<launch>
<arg name="model" />
<param name="robot_description" textfile="$(find robotmodel)/urdf/eros.urdf" />
<param name="use_gui" value="true"/>
<node name="model_publisher" pkg="robotmodel"
type="model_publisher" />
<node name="robot_state_publisher" pkg="robot_state_publisher"
type="state_publisher" />
<node name="rviz" pkg="rviz" type="rviz" args="-d $(find robotmodel)/rviz/urdf_odom.rviz" required="true" />
</launch>
